src/init/features/dataconnect/index.ts

```diff
@@ -69,116 +218,46 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
       });
     }
 
-    newInstance = true;
-    cloudSqlInstanceId = await promptOnce({
-      message: `What ID would you like to use for your new CloudSQL instance?`,
-      type: "input",
-      default: `fdc-sql`,
-    });
-    locationId = await promptOnce({
+    info.locationId = await promptOnce({
       message: "What location would you use for this instance?",
```

Comment:
nit: This is both the location for FDC service and CSQL instance.

It's only needed for new service and new instance.
Maybe say "what location would you use?"

Reply:
Done.